aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -252,10 +264,27 @@ def _construct_builder(self, **kwargs) -> engine.ProcessBuilder:
         ## Redefining stress threshold.
         if threshold_stress is not None:
             parameters['MOTION']['CELL_OPT']['PRESSURE_TOLERANCE'] = f'[GPa] {threshold_stress * EV_A3_TO_GPA}'
+
+        ## Lowering runtime by 5 minutes to let CP2K gracefully finish the calculation.
+        walltime = engines['relax']['options']['max_wallclock_seconds']
+        walltime = max(300, walltime - 300)
```

Comment:
Can we add CP2K specific parameters such that a user using the common workflows can override it easily?

Reply:
I would say this is for a different PR. The big idea of ACWF is having the same input for different codes. I am not sure whether code-fine-tuning options are welcome at all.

Maybe @sphuber can comment on this.